terms/demographics/reportedGender.json

```diff
@@ -0,0 +1,20 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema",
+  "$id": "sage.annotations-demographics.reportedGender-0.0.1",
```

Comment:
Is this needed? We have 'sex' as a term and I would expect people to use that term, regardless of whether the tissue was from a bank or not. I could see a term for 'reportedGender' if the subject was alive and was self-reporting their gender, but less so for this case.